### PR TITLE
[codestlye] xml site non components

### PR DESCRIPTION
**Pull request by brianteeman**
codestyle changes for the files not included in the other PR

### CODE REVIEW

File: plugins/fields/list/params/list.xml [hunk 1]
```diff
@@ -7,31 +7,34 @@
...
+				multiple="true"
+			/>
 				<form hidden="true" name="list_templates_modal" repeat="true">
 					<field
 						name="name"
+						type="text"
 						label="PLG_FIELDS_LIST_PARAMS_OPTIONS_NAME_LABEL"
 						size="30"
-						type="text"
 					/>
```
Comment (wojsmol) [inline plugins/fields/list/params/list.xml]: `</field>` instead of `/>`

File: plugins/fields/list/params/list.xml [hunk 2]
```diff
@@ -7,31 +7,34 @@
...
 
 					<field
 						name="value"
+						type="text"
 						label="PLG_FIELDS_LIST_PARAMS_OPTIONS_VALUE_LABEL"
 						size="30"
-						type="text"
 					/>
```
Comment (wojsmol) [inline plugins/fields/list/params/list.xml]: `</field>` instead of `/>`

File: plugins/fields/media/media.xml [hunk 3]
```diff
@@ -24,21 +24,21 @@
...
 					name="directory"
 					type="folderlist"
+					label="PLG_FIELDS_MEDIA_PARAMS_DIRECTORY_LABEL"
+					description="PLG_FIELDS_MEDIA_PARAMS_DIRECTORY_DESC"
 					directory="images"
 					hide_none="true"
 					recursive="true"
-					label="PLG_FIELDS_MEDIA_PARAMS_DIRECTORY_LABEL"
-					description="PLG_FIELDS_MEDIA_PARAMS_DIRECTORY_DESC"
 				/>
```
Comment (wojsmol) [inline plugins/fields/media/media.xml]: `</field>` instead of `/>`

File: plugins/fields/radio/params/radio.xml [hunk 4]
```diff
@@ -5,25 +5,25 @@
...
 				type="subform" 
+				label="PLG_FIELDS_RADIO_PARAMS_OPTIONS_LABEL"
+				description="PLG_FIELDS_RADIO_PARAMS_OPTIONS_DESC"
 				layout="joomla.form.field.subform.repeatable-table"
 				icon="list" 
 				multiple="true"
-				description="PLG_FIELDS_RADIO_PARAMS_OPTIONS_DESC"
-				label="PLG_FIELDS_RADIO_PARAMS_OPTIONS_LABEL"
-			>
+			/>
```
Comment (wojsmol) [inline plugins/fields/radio/params/radio.xml]: `</field>` instead of `/>`

File: plugins/fields/radio/params/radio.xml [hunk 5]
```diff
@@ -5,25 +5,25 @@
...
 				<form hidden="true" name="list_templates_modal" repeat="true">
 					<field
 						name="name"
+						type="text"
 						label="PLG_FIELDS_RADIO_PARAMS_OPTIONS_NAME_LABEL"
 						size="30"
-						type="text"
 					/>
```
Comment (wojsmol) [inline plugins/fields/radio/params/radio.xml]: `</field>` instead of `/>`

File: plugins/fields/radio/params/radio.xml [hunk 6]
```diff
@@ -5,25 +5,25 @@
...
 
 					<field
 						name="value"
+						type="text"
 						label="PLG_FIELDS_RADIO_PARAMS_OPTIONS_VALUE_LABEL"
 						size="30"
-						type="text"
 					/>
```
Comment (wojsmol) [inline plugins/fields/radio/params/radio.xml]: `</field>` instead of `/>`

**Comment by wojsmol** [comment]
I have tested this item :white_check_mark: successfully on 4967ac2a32fc7b58fbd539f694a9eb7a0fd52190<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15952">issues.joomla.org/tracker/joomla-cms/15952</a>.</sub>

File: plugins/content/emailcloak/emailcloak.xml [hunk 7]
```diff
@@ -22,10 +22,10 @@
 				<field
 					name="mode"
 					type="list"
-					default="1"
 					description="PLG_CONTENT_EMAILCLOAK_MODE_DESC"
```
Comment (Quy) [inline plugins/content/emailcloak/emailcloak.xml]: Swap lines

**Comment by tonypartridge** [comment]
I have tested this item :white_check_mark: successfully on 390fea5022d3c73da92848ba0347ecdbe6e013e3<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15952">issues.joomla.org/tracker/joomla-cms/15952</a>.</sub>

**Comment by franz-wohlkoenig** [comment]
@wojsmol can you please retest?

File: modules/mod_articles_popular/mod_articles_popular.xml [hunk 8]
```diff
@@ -38,9 +38,9 @@
 				<field
 					name="count"
 					type="text"
-					default="5"
 					label="MOD_POPULAR_FIELD_COUNT_LABEL"
 					description="MOD_POPULAR_FIELD_COUNT_DESC"
+					default="5"
 				/>
 
 					<field
```
Comment (wojsmol) [inline modules/mod_articles_popular/mod_articles_popular.xml]: indent

Comment (Quy) [inline modules/mod_articles_popular/mod_articles_popular.xml]: and lines 46 and 128

**Comment by rdeutz** [comment]
merged on review

File: modules/mod_articles_popular/mod_articles_popular.xml (hunk 8 above)
Comment (Quy) [inline modules/mod_articles_popular/mod_articles_popular.xml]: @brianteeman ping just in case these have not been fixed in another pr.

**Comment by brianteeman** [comment]
thanks

**Pull request merged**